pkg/rest/utils.go

```diff
@@ -2,30 +2,25 @@ package rest
 
 import (
 	"fmt"
+	"regexp"
 	"strings"
 )
 
+var multiSlashRegex = regexp.MustCompile("[/]+")
+
 func sanitizePath(route string) string {
+	route = fmt.Sprintf("/%s", route)
+	route = multiSlashRegex.ReplaceAllString(route, "/")
```

Comment:
We now have a different approach to sanitize:
* we add a prefix "/"
* we replace all occurrences of a multi-slash by a single one
* in case the end path is empty, we return "/"

This is much simpler.